### PR TITLE
Feat - chart types

### PR DESCRIPTION
**Pull request by Int-Angel**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Added the 3 types of charts

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: data-dashboard/lib/components/Chart/Chart.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-unused-vars */
+import { BarChart, LineChart, DonutChart } from '@tremor/react';
+import ChartCard from '../ChartCard/ChartCard';
+
+import 'react-grid-layout/css/styles.css';
+import 'react-resizable/css/styles.css';
+
+export enum ChartType {
```
Comment (antonieto) [inline data-dashboard/lib/components/Chart/Chart.tsx]:
```suggestion
export type ChartType = 'bar' | 'line' | 'pie'
```

File: data-dashboard/lib/components/Chart/Chart.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  bar = 'bar',
+  line = 'line',
+  pie = 'pie',
+}
+
+interface Props {
+  type: string;
+  title: string;
+  chartProps: any;
```
Comment (antonieto) [inline data-dashboard/lib/components/Chart/Chart.tsx:12]: This is very dangerous, please find a better way to define `chartProps`.

Comment (antonieto) [reply data-dashboard/lib/components/Chart/Chart.tsx:12]: This might work.
```suggestion
  chartProps: React.ComponentProps<typeof BarChart>;
```

**Review by antonieto — CHANGES_REQUESTED** [review]

**Review by antonieto — CHANGES_REQUESTED** [review]

**Review by antonieto — DISMISSED** [review]

**Review by paotg123 — APPROVED** [review]

**Review by paotg123 — DISMISSED** [review]
👍🏼

File: yarn.lock
Comment (antonieto) [inline yarn.lock:1]: Se coló este file prime

**Review by antonieto — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (Int-Angel) [reply yarn.lock:1]: alv

**Review by DavBot02 — APPROVED** [review]

**Review by antonieto — APPROVED** [review]

**Pull request merged**